content/docs/for-buildpack-authors/how-to/write-buildpacks/specify-env.md

```diff
@@ -3,15 +3,108 @@ title="Specify the environment"
 weight=99
 +++
 
+Environment variables are a common way to configure various buildpacks at build-time.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+### POSIX Path Variables
+
+The following layer path environment variables MUST be set by the `lifecycle` during the `build` and `launch` phases in order to make buildpack dependencies accessible.
+
+During the `build` phase, each variable designated for build MUST contain absolute paths of all previous buildpacks’ `<layers>/<layer>/` directories that are designated for build.
+
+When the exported OCI image is launched, each variable designated for launch MUST contain absolute paths of all buildpacks’ `<layers>/<layer>/` directories that are designated for launch.
+
+In either case,
+
+- The lifecycle MUST order all `<layer>` paths to reflect the reversed order of the buildpack group.
+- The lifecycle MUST order all `<layer>` paths provided by a given buildpack alphabetically ascending.
+- The lifecycle MUST separate each path with the OS path list separator (e.g. `:` on Linux, `;` on Windows).
+
+| Env Variable                               | Layer Path   | Contents         | Build | Launch |
+|--------------------------------------------|--------------|------------------|-------|--------|
+| `PATH`                                     | `/bin`       | binaries         | [x]   | [x]    |
+| `LD_LIBRARY_PATH`                          | `/lib`       | shared libraries | [x]   | [x]    |
+| `LIBRARY_PATH`                             | `/lib`       | static libraries | [x]   |        |
+| `CPATH`                                    | `/include`   | header files     | [x]   |        |
+| `PKG_CONFIG_PATH`                          | `/pkgconfig` | pc files         | [x]   |        |
+
+The platform SHOULD NOT assume any other base-image-provided environment variables are inherited by the buildpack.
+
+### Build Environment Variables
+
+During the `build` phase, buildpacks MAY write environment variable files to `<layers>/<layer>/env/`, `<layers>/<layer>/env.build/`, and `<layers>/<layer>/env.launch/` directories.
+
+For each `<layers>/<layer>/` designated as a build layer, for each file written to `<layers>/<layer>/env/` or `<layers>/<layer>/env.build/` by `/bin/build`, the `lifecycle` MUST modify an environment variable in subsequent executions of `/bin/build` according to the [environment variable modification rules](https://github.com/buildpacks/spec/blob/main/buildpack.md#environment-variable-modification-rules).
+
+For each file written to `<layers>/<layer>/env/` or `<layers>/<layer>/env.launch/` by `/bin/build`, the `lifecycle` MUST modify an environment variable during the launch phase according to the modification rules above.
+
+### Process Environment Variables
```

Comment:
Do we have a concept of "Process Environment Variables"?  I'm aware of build-time env vars and runtime env vars.

Reply:
Yep there are process-specific environment variables. I think if we show a directory tree showing all the different ways of configuring the environment that will make it clearer.